File: BubbleSort/bubbleSort.py
def BubbleSort(arr:list,n:int):
        for i in range(0,n):
            for j in range(0,n-i-1):
                if arr[j] > arr[j+1]:
                    aux = arr[j]
                    arr[j] = arr[j+1]
                    arr[j+1] = aux

def run():
    a = input("")
    A = [14,13,11,10]
    n = len(A)
    print(A)
    BubbleSort(A,n)
    print(A)

if __name__ == '__main__':
    run()

File: BubbleSort/bubbleSort.c
#include <stdio.h>

void BubbleSort(int*,int);

int main(){
    int A[] = {14,13,11,10};
    int n = sizeof(A) / sizeof(A[0]);
    for(int i=0; i<n; i++){
        printf("%d ",A[i]);
    }
    printf("\n");
    BubbleSort(A,n);
    for(int i=0; i<n; i++){
        printf("%d ",A[i]);
    }
}
void BubbleSort(int* arr,int n){
    int i,j,aux;
    for(i=0; i<n; i++){
        for(j=0; j<n; j++){
            if(arr[j] > arr[j+1]){
                aux = arr[j];
                arr[j] = arr[j+1];
                arr[j+1] = aux;
            }
        }
    }
}